src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,77 @@
-import React from 'react';
+import React, { useState } from 'react';
+import classNames from 'classnames';
 import './FindMovie.scss';
+import { getMovie } from '../../api';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
+
+type Props = {
+  movies: Movie[];
+  setMovies: (value: Movie[]) => void;
+};
+
+export const FindMovie: React.FC<Props> = ({
+  movies, setMovies,
+}) => {
+  const [movie, setMovie] = useState<Movie | null>(null);
+  const [title, setTitle] = useState('');
+  const [loading, setLoading] = useState(false);
+  const [isError, setIsError] = useState(false);
+
+  const handleQueryChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setTitle(event.target.value);
+    setIsError(false);
+  };
+
+  const handleAddClick = () => {
+    if (movie && !movies.some(m => m.imdbId === movie.imdbId)) {
+      setMovies([...movies, movie]);
+    }
+
+    setMovie(null);
+    setTitle('');
+  };
+
+  const handleFormSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    setLoading(true);
+
+    getMovie(title)
+      .then((response) => {
+        if ('Error' in response) {
+          setIsError(true);
+
+          return;
+        }
+
+        const {
+          Title, Plot, Poster, imdbID,
+        } = response;
+
+        const newMovie: Movie = {
+          title: Title,
+          description: Plot,
+          imgUrl: Poster !== 'N/A' ? Poster
+            : 'https://via.placeholder.com/360x270.png?text=no%20preview',
```

Comment:
Extract the placeholder URL to a constant. This improves readability and maintainability of the code.